Tests/TuistPluginTests/PluginServiceTests.swift

```diff
@@ -0,0 +1,134 @@
+import TSCBasic
+import TuistCore
+import TuistLoader
+import TuistLoaderTesting
+import TuistSupport
+import TuistSupportTesting
+import XCTest
+
+@testable import TuistPlugin
+
+final class PluginServiceTests: TuistTestCase {
+    private var mockModelLoader: MockGeneratorModelLoader!
+    private var mockGitHandler: MockGitHandler!
+    private var mockManifestFilesLocator: MockManifestFilesLocator!
+    private var subject: PluginService!
+
+    override func setUp() {
+        super.setUp()
+        do {
+            let tuistPath = try temporaryPath().appending(RelativePath("Tuist"))
+            mockModelLoader = MockGeneratorModelLoader(basePath: tuistPath)
+            mockGitHandler = MockGitHandler()
+            mockManifestFilesLocator = MockManifestFilesLocator()
+            subject = PluginService(
+                modelLoader: mockModelLoader,
+                fileHandler: fileHandler,
+                gitHandler: mockGitHandler,
+                manifestFilesLocator: mockManifestFilesLocator
+            )
+        } catch {
+            XCTFail("Failed initializing PluginServiceTests")
+        }
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    func test_loadPlugins_atPath_WHEN_localHelpers() throws {
```

Comment:
In most test cases we follow the `// Given // When // Then` convention, so I'd add that to the test cases in this file.